go-controller/pkg/util/util.go

```diff
@@ -329,40 +331,52 @@ func IsClusterIP(svcVIP string) bool {
 	return false
 }
 
-// GetActiveNetworkForNamespace returns the active network for the given namespace
-// based on the NADs present in that namespace.
+func defaultNetConf() *ovncnitypes.NetConf {
+	return &ovncnitypes.NetConf{NetConf: cnitypes.NetConf{Name: types.DefaultNetworkName}}
```

Comment:
util.NewNetInfo(
&ovncnitypes.NetConf{
NetConf: cnitypes.NetConf{Name: types.DefaultNetworkName},
},
)

maybe put this util closer to there so that people know it exists..

Reply:
That returns a NetInfo I need a NetConf also it is not worth it to handler errors from `NewNetInfo` in this case.